packages/medusa/src/services/totals.ts

```diff
@@ -208,14 +208,7 @@ class TotalsService extends TransactionBaseService {
     }
 
     if (opts.include_tax) {
-      if (isOrder(cartOrOrder) && cartOrOrder.tax_rate !== null) {
-        totals.original_tax_total = Math.round(
-          totals.original_tax_total * (cartOrOrder.tax_rate / 100)
-        )
-        totals.tax_total = Math.round(
-          totals.original_tax_total * (cartOrOrder.tax_rate / 100)
-        )
-      } else {
+      if (isCart(cartOrOrder) || cartOrOrder.tax_rate === null) {
```

Comment:
question: I am wondering if it really matters to know if it is a cart or an order in the statements as it does not change the logic